SD_Card.h

```diff
@@ -0,0 +1,85 @@
+#include <SD.h>
+#include <SPI.h>
+
+#include "gps.h"
+#include "imu.h"
+
+#pragma once
+
+class SDCard {
+  private:
+    File m_sdCardDataFile;
+    static const int m_chipSelect = 4; // this is not the final value - SDCARD_SS_PIN is the correct one for the MKR Zero
+    static const String m_fileName = "rocketry_data";
+  public:
+    void writeToCSV( // this is equivalent 
+      const GPS::Coordinates& coords,
+      const IMU::vector3& accel,
+      const IMU::vector3& gyro,
+      float altitude,
+      int analogReading, // this is VOC
+      float humidity,
+      float temperature) {
```

Comment:
Since you're including values for the VOC, temp, and humidity, I will assume this code will work for Capsule 2, but not for Capsule 1. @bbrk24 are we able to write SD card code that is able to accommodate both capsules in the same file? Ie, is there something that can be done here to make this general enough for both capsules?

Reply:
The fix is to simply wrap the Capsule 2-specific parts of the code in `#if CAPSULE == 2`. I defined that variable specifically to make things like this easier. I already do this in parts of frame.h -- though I forgot to do it for the parameter list to `createFrame` at first, I fixed that in #5.

Reply:
awesome, Cam please do this for the SD card write code as well

Reply:
Just fixed those changes - can you take a look, please?

Reply:
Looks like you did it correctly in the body of the function, but the parameters should also be `#if`d out: https://github.com/bbrk24/BSC_Arduino/blob/392f541522fee0b189430039aa29906674bbd708/frame.h#L87-L93